src/Provider/MetadataInterface.php

```diff
@@ -13,27 +13,55 @@
 
 namespace Sonata\MediaBundle\Provider;
 
-/**
- * NEXT_MAJOR: Remove CoreBundle dependency.
- */
-interface MetadataInterface extends \Sonata\CoreBundle\Model\MetadataInterface
-{
-    public function getTitle(): string;
+if (interface_exists('Sonata\CoreBundle\Model\MetadataInterface')) {
+    /**
+     * NEXT_MAJOR: Remove CoreBundle dependency.
+     */
+    interface MetadataInterface extends \Sonata\CoreBundle\Model\MetadataInterface
```

Comment:
What is the usage of this Metadata? According to CoreBundle you should extend the one on BlockBundle

Reply:
I don't know.

Reply:
Anyway decprecation message tell US we can use BlockBundle matsdata instead.

Reply:
> What is the usage of this Metadata? According to CoreBundle you should extend the one on BlockBundle

That's not right. There are different Metadatas: Block information metadata and media provider metadata.

Reply:
I was looking at this comment:  @deprecated since sonata-project/core-bundle 3.13.0, to be removed in 4.0. Use Sonata\BlockBundle\Meta\MetadataInterface instead.

But I dont really know what is the use for MetadataInterface

Reply:
i think this is another reason to create `sonata-project/contracts`

Reply:
Well, maybe we don't have to extend the block bundle one, I was just following the comments on the code. But it is correct that it doesn't have much sense

Reply:
Metadata is generated for each media by provider. It have sense for `MediaBlockService` but also make it more confusing.
For me we should keep MediaMetadata and Block Metadata seperate but based on `contracts/metadata`. This will keep desteny in naming and allow use metadata in other places.